### PR TITLE
Fix Property Naming Issues

### PR DESCRIPTION
**Pull request by Linfye**
### Contributor checklist

-   [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This PR fixed ktlint issues of property naming.
### Related issue

-   #84

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for this, @Linfye! Will check it later :) Very cool to have an example for other issues to work from to solve more linting problems 😊

**Review by angrezichatterbox — APPROVED** [review]
PR looks fine to me. Thank you for the contributions @Linfye. :)

File: .editorconfig [hunk 1]
```diff
@@ -20,7 +20,7 @@ continuation_indent_size = 4
 wildcard_import_limit = 999
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
-ktlint_standard_property-naming = disabled
+#ktlint_standard_property-naming = disabled
```
Comment (andrewtavis) [inline .editorconfig:23]: For the disabled ones, let's just delete them in the future :)

File: app/src/main/java/be/scri/activities/BaseSimpleActivity.kt [hunk 2]
```diff
@@ -393,12 +392,12 @@ abstract class BaseSimpleActivity : AppCompatActivity() {
             } else {
                 funAfterSAFPermission?.invoke(false)
             }
-        } else if (requestCode == DELETE_FILE_SDK_30_HANDLER) {
+        } else if (requestCode == Companion.DELETE_FILE_SDK_30_HANDLER) {
```
Comment (andrewtavis) [inline app/src/main/java/be/scri/activities/BaseSimpleActivity.kt:395]: Nice :) Great work and thoughts on this, both of you!

**Review by andrewtavis — APPROVED** [review]
Really appreciate you picking this up, @Linfye! Really can't wait until we can look at the codebase and all of the ktlint/detekt issues are done! Gonna be great ✨

File: .editorconfig (hunk 1 above)
Comment (Linfye) [reply .editorconfig]: I'll follow this in the next linting issue and fix this one then. Thanks for your review.😊

**Pull request merged**